R/plot-construction.r

```diff
@@ -38,6 +38,12 @@
 #' # This can be useful to return from a function.
 #' base + list(subset(mpg, fl == "p"), geom_smooth())
 "+.gg" <- function(e1, e2) {
+  if (missing(e2)) {
+    stop("There's nothing on the left-hand side of `+`. ",
```

Comment:
We need to tweak this slightly to follow the tidyverse error style guide (and to make the hint slightly more helpful). Maybe:

Can not use `+.gg` with a single argument. Did you accidentally put `+` on a new line?